uftrace.c

```diff
@@ -985,6 +990,7 @@ int main(int argc, char *argv[])
 		.depth		= OPT_DEPTH_DEFAULT,
 		.max_stack	= OPT_RSTACK_DEFAULT,
 		.port		= UFTRACE_RECV_PORT,
+		.pid		= 0,
```

Comment:
You don't need to zero-initialize here - that's the thing the compiler will do for us.